x-pack/plugins/snapshot_restore/public/app/sections/home/repository_list/repository_details/repository_details.tsx

```diff
@@ -147,6 +149,30 @@ const RepositoryDetailsUi: React.FunctionComponent<Props> = ({
           </EuiFlexItem>
         </EuiFlexGroup>
         <EuiSpacer size="l" />
+        <EuiTitle size="s">
+          <h3>
+            <FormattedMessage
+              id="xpack.snapshotRestore.repositoryDetails.snapshotsTitle"
+              defaultMessage="Snapshots"
+            />
+          </h3>
+        </EuiTitle>
+        <EuiSpacer size="s" />
+        {Number.isInteger(snapshots.count) ? (
+          <EuiLink href={linkToSnapshots(repositoryName)}>
+            <FormattedMessage
+              id="xpack.snapshotRestore.repositoryDetails.snapshotsDescription"
+              defaultMessage="{count} {count, plural, one {snapshot} other {snapshots}} found"
+              values={{ count: snapshots.count }}
+            />
+          </EuiLink>
+        ) : (
+          <FormattedMessage
+            id="xpack.snapshotRestore.repositoryDetails.noSnapshotsDescription"
+            defaultMessage="No snapshot information"
```

Comment:
If count is 0, can we show a message saying "Repository contains no snapshots"?

Reply:
see above!